lib/boot.js

```diff
@@ -56,10 +56,14 @@ const client = initClient({
         return {
           ...note,
           contentKey: content
+            .replace(/^\s*#+\s*/, '') // Remove leading whitespace and Markdown heading
```

Comment:
Should we include more Markdown characters here like `*`, `-`, `_` etc?

Reply:
I think this is fine for now, since we're only stripping the `#` chars for the [Note List titles](https://github.com/Automattic/simplenote-electron/blob/a5cc00443c0f89719912ab69ff8c9334e8da01be/lib/utils/note-utils.js#L15), and the sorting should reflect the text that's displayed in the Note List. (We can consider doing more aggressive Markdown stripping in both the index and Note List titles if we get a lot of requests!)